### PR TITLE
zuul.yaml: run some checks after installation

### PR DESCRIPTION
**Pull request by fredericlepied**

### CODE REVIEW

File: pre.yaml [hunk 1]
```diff
@@ -6,3 +6,13 @@
...
       args:
         chdir: "{{ zuul.project.src_dir }}"
+    - name: check packages
+      become: yes
+      command: "apt-get install -fs -qq"
+      args:
+        chdir: "{{ zuul.project.src_dir }}"
+    - name: check services
+      become: yes
+      command: "wazo-service status"
```
Comment (sduthil) [inline pre.yaml]: Currently, `wazo-service status` is not expected to report OK for every service. The engine needs to be setup via wazo-setupd first.

Comment (fredericlepied) [reply pre.yaml]: how can I do that in ansible ?

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
LGTM

**Pull request merged**